docs/website/docs/building-from-source/python-bindings-and-importers.md

```diff
@@ -1,19 +1,24 @@
-# Optional features
+# Python bindings and importers
 
-This page details the optional features and build modes for the project.
-Most of these are controlled by various CMake options, sometimes requiring
-extra setup or preparation. Each section extends the basic build steps
-in the [getting started](./getting-started.md) page.
+!!! Attention
+    These components are more complicated to build from source than the rest of
+    the project. If your usage does not require making source changes, prefer
+    to install from the official binary distributions instead.
```

Comment:
Add the link to `../bindings/python.md` as well?

Reply:
Good idea, done.